packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -863,54 +864,34 @@ class SlideshowView extends GalleryComponent {
   createDebugMsg() {
     return <GalleryDebugMessage {...this.props.debug} />;
   }
-
-  getArrowsRenderData() {
-    const { customNavArrowsRenderer } = this.props.customComponents;
-    const { arrowsSize } = this.props.options;
-    if (customNavArrowsRenderer) {
-      return {
-        arrowRenderer: customNavArrowsRenderer,
-        navArrowsContainerWidth: arrowsSize,
-        navArrowsContainerHeight: arrowsSize
-      }
+  
+  getShouldRenderArrowsArgs(){
```

Comment:
`getShouldRenderArrowsArgs` is arrows related, please move the the `navigationArrowUtils` file